status/wallet2/account.nim

```diff
@@ -79,8 +79,9 @@ proc `$`*(self: OpenseaAsset): string =
 
 proc getOpenseaCollectionTraits*(jsonCollection: JsonNode): Table[string, OpenseaCollectionTrait] =
     var traitList: Table[string, OpenseaCollectionTrait] = initTable[string, OpenseaCollectionTrait]()
-    for key, value in jsonCollection{"traits"}:
-        traitList[key] = OpenseaCollectionTrait(min: value{"min"}.getFloat, max: value{"max"}.getFloat)
+    if jsonCollection.hasKey("traits"):
```

Comment:
I don't think you need that fix, what was needed is the update of status-go. Could you retry once this one has been merged?

https://github.com/status-im/status-desktop/pull/3587

Reply:
Master isn't working currently without this change and #3587 will work with or without this change. Been trying to get a PR merged in Status Desktop for 9 days lol, and would prefer to not have to keep rebasing it if possible.